src/initial-submission/components/Routes.tsx

```diff
@@ -2,10 +2,11 @@ import React from 'react';
 import { Switch } from 'react-router-dom';
 import SubmissionWizard from './SubmissionWizard';
 import AuthRoute from '../../core/components/AuthRoute';
+import ThankYouPage from './ThankYouPage';
 
 const Routes: React.FC = (): JSX.Element => (
     <Switch>
-        <AuthRoute path="/thankyou/:id" component={(): JSX.Element => <div>Thankyou page</div>} />
+        <AuthRoute path="/thankyou/:id" component={ThankYouPage} />
```

Comment:
Could this be movedto the same place that the Survey component is loaded in at (top level `core/components/App.tsx`

Reply:
actually not sure this is appropriate as the thankyou page is part of the wizard 🤔